go.mod

```diff
@@ -23,4 +23,5 @@ require (
 	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
 	google.golang.org/api v0.71.0
 	gopkg.in/ini.v1 v1.66.4
+	gotest.tools/gotestsum v1.7.0 // indirect
```

Comment:
Please revert changes to go.mod and go.sum - probably related to running go get from the same folder.

Reply:
Done.